pkg/rtc/participant.go

```diff
@@ -576,17 +579,20 @@ func (p *ParticipantImpl) createPublsiherAnswerAndSend() error {
 		return errors.Wrap(err, "could not create answer")
 	}
 
-	p.params.Logger.Infow("sending pub answer (unfiltered)", "state", p.State().String(), "sdp", answer.SDP)
-	answer = p.publisher.FilterCandidates(answer)
-	p.params.Logger.Infow("sending pub answer (filtered)", "state", p.State().String(), "sdp", answer.SDP)
-
+	p.publisher.Logger().Infow("local answer (unfiltered)", "sdp", answer.SDP)
 	if err = p.publisher.pc.SetLocalDescription(answer); err != nil {
 		prometheus.ServiceOperationCounter.WithLabelValues("answer", "error", "local_description").Add(1)
 		return errors.Wrap(err, "could not set local description")
 	}
 
-	p.params.Logger.Debugw("sending answer to client")
-
+	//
+	// Filter after setting local description as pion expects the answer
+	// to match between CreateAnswer and SetLocalDescription.
+	// Filtered answer is sent to remote so that remote does not
+	// see filtered candidates.
+	//
+	answer = p.publisher.FilterCandidates(answer)
+	p.publisher.Logger().Infow("local answer (filtered)", "sdp", answer.SDP)
```

Comment:
let's remove these?

Reply:
I added these logs to be able to debug. Will remove them once it has gotten some soak.

Reply:
got it, that makes sense. can we reduce log spam by only logging when preferTCP is enabled? wdyt?

Reply:
Yeah, can restrict to that for now.